packages/webpack/mixins/config/mixin.core.js

```diff
@@ -60,9 +65,39 @@ class WebpackConfigMixin extends Mixin {
 }
 
 WebpackConfigMixin.strategies = {
-  getBuildConfig: callable,
-  collectBuildConfigs: sequence,
-  configureBuild: sequence,
+  getBuildConfig: validate(callable, ([target, baseConfig]) => {
+    invariant(
+      typeof target === 'string',
+      'getBuildConfig(): Received invalid target string'
+    );
+    invariant(
+      !baseConfig || typeof baseConfig === 'string',
+      'getBuildConfig(): Received invalid baseConfig string'
```

Comment:
In my opinion this message is quite vague too - can we make it more descriptive?